solutions/kubernetes/ceph-binary-build.sh

```diff
@@ -61,7 +61,7 @@ function prereq() {
 
     pushd "$BUILD_LOCATION"
         add_common_separator "Removing previous files"
-        rm -rvf *
+        rm -rvf ./*
```

Comment:
Let me know if we are good to make this kind of changes suggested by codacy. Link : https://app.codacy.com/gh/Seagate/cortx-re/file/71107647137/issues/source?bid=27005559&fileBranchId=27005559#l64

Reply:
Please check that scripts works with this change